frontend/src/index.css

```
@@ -1,6 +1,9 @@
 @import "tailwindcss";
+@import "tw-animate-css";
 @import "sonner/dist/styles.css";
 
+@custom-variant dark (&:is(.dark *));
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)